frontend/components/table/ProductTableRow.tsx

```diff
@@ -16,7 +16,9 @@ type Props = {
 const ProductTableRow = ({ product }: Props) => {
   const bbContentContainer = React.useRef<HTMLDivElement | null>(null);
   const [numberOfHidenBBImages, setNumberOfHidenBBImages] = useState<number>(0);
-  const [imageSectionWidth, setImageSectionWidth] = useState<string | undefined>();
+  const [imageSectionWidth, setImageSectionWidth] = useState<
+    string | undefined
+  >();
```

Comment:
I think <string> is enough, since if length is 0 it should return false, so the style below should apply correctly.

Reply:
But in the beginning it is undefined this is why I have this type

Reply:
what if you put `''` empty string inside () ? :sweat_smile:

Reply:
Because I use it here:
<img width="537" alt="Zrzut ekranu 2023-11-7 o 11 35 26" src="https://github.com/GovStackWorkingGroup/testing-webapp/assets/80507519/1f79219d-0d37-4c33-8b92-8722c7a38022">

And "" ?? "75%" -> "" not "75%"